exs/mundo_1/c#/031.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+/*
+	031 - Custo da viagem
+  	Desenvolva um programa que pergunte a distância de uma viagem em Km.
+	Calcule o preço da passagem, cobrando R$ 0,50 por Km para viagens de até 200km e R$ 0,45 para viagens mais longas
+*/
+					
+public class Program
+{
+	public static void Main()
+	{
+		Console.Write("Digite a distancia da viagem em km: ");
+		float km = float.Parse(Console.ReadLine());
+		float valorTotal;
+		if (km < 200)
```

Comment:
Note que o operador lógico que deve ser usado é `<=`, "[...] **até** 200km e [...]".
```suggestion
		if (km <= 200)
```